os/include/tinyara/mm/mm.h

```diff
@@ -694,6 +694,8 @@ struct mm_heap_s *mm_get_heap_with_index(int index);
 
 int mm_get_heapindex(void *mem);
 
+size_t mm_get_largest_freenode_size(void *address);
```

Comment:
I think its not a good idea to pass address to this API. We just want to know the largest free node in the heap. So, why do we need address? Also, if we want to pass an address, we need to have a previous malloc pointer. So, I think this is not a convenient way to call this API.

Reply:
Yes, we can get kheap without address, but we don't have any way to get the user heap without address.
Do you have any idea?